.github/workflows/nightvision.yml

```diff
@@ -19,20 +21,26 @@ jobs:
       
       - name: (2) Install NightVision
         run: |
-            wget -c https://downloads.nightvision.net/binaries/latest/nightvision_latest_linux_amd64.tar.gz -O - | tar -xz; sudo mv nightvision /usr/local/bin/
-                    python -m pip install semgrep --user
+          wget -c https://downloads.nightvision.net/binaries/latest/nightvision_latest_linux_amd64.tar.gz -O - | tar -xz; sudo mv nightvision /usr/local/bin/
+          python -m pip install semgrep --user
 
       - name: (3) Extract API documentation from code
-        run: nightvision swagger-extract ./ -u ${NIGHTVISION_TARGET} --lang spring
+        run: |
+          nightvision swagger extract ./ -t ${NIGHTVISION_TARGET} --lang spring || true
```

Comment:
This workaround insures the pipeline will not fail to scan if the `swagger extract` fails for any reason.